functional-samples/libraries-xhr-in-sw/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Fetching Titles",
+  "version": "0.1",
+  "manifest_version": 3,
+  "description": "This extension fetches the titles of all the tabs in the current window and displays them in a side panel.",
+  "background": {
+    "service_worker": "dist/background.js"
+  },
+  "permissions": ["activeTab", "scripting", "sidePanel"],
```

Comment:
Do you need scripting permission?

Reply:
nope! changed the demo and forgot to remove this. good catch!